standard/table.lua

```diff
@@ -249,6 +249,34 @@ function Table.iter.spairs(tbl, order)
 	end
 end
 
+--[[
+Iterates over table entries whose keys are prefixed numbers. The entries are
+visited in order, starting from 1. The iteration stops upon a skipped number.
+
+Example:
+local args = {
+	p1 = {},
+	p2 = {},
+	p3 = {},
+	foo = {},
+	p10 = {},
+}
+for key, player in Table.iter.prefix(args, 'p') do
+	mw.log(key)
+end
+
+will print out 'p1 p2 p3'
+]]
+function Table.iter.prefix(tbl, prefix)
```

Comment:
```suggestion
function Table.iter.forEachByPrefix(tbl, prefix)
```

?

Reply:
`Table.iter.prefix` is a function that returns an iterator (like pairs/ipairs), which makes it different from `forEach` which takes in a callback as its 2nd parameter.

Reply:
I'm using a foreach variant in `StarcraftParticipantTable.forPlayerKeys` here: https://liquipedia.net/commons/Module:ParticipantTable/Starcraft

Reply:
`pairsByPrefix`?

Reply:
ok, renamed to `pairsByPrefix`